acme/src/main/java/io/micronaut/acme/events/CertificateEvent.java

```diff
@@ -24,6 +24,7 @@
 public class CertificateEvent {
     private final X509Certificate certificate;
```

Comment:
Is this still required, ie. should there be a distinction between "single certificate" and "certificate chain"? 
The "single certificate" code path is only used for tls-apln-01 validation, but this could also utilize the `fullCertificateChain` array (just with a single entry).

Reply:
Yah thats fair. I left it due to the CertificateEvent constructor is public so in theory someone could be using it. But yes implementation wise I could just store the full chain and in the validation cert case an array of 1 instead of 2 objects.